src/renderers/dom/shared/CSSPropertyOperations.js

```diff
@@ -230,15 +230,17 @@ var CSSPropertyOperations = {
       if (__DEV__) {
         warnValidStyle(styleName, styles[styleName], component);
       }
+      var isCustomProperty = styleName.indexOf('--') === 0;
       var styleValue = dangerousStyleValue(
         styleName,
         styles[styleName],
         component,
+        isCustomProperty,
```

Comment:
This is not the only place this function is called as far as I can see. Can you please make sure this argument always exists?